src/App.php

```diff
@@ -133,11 +141,11 @@ public function dispatch(): ResponseInterface
         switch ($routeInfo[0]) {
```

Comment:
Unfortunately, I cannot use the array destructuring feature of PHP since the `dispatch` or FastRoute can return an array with one, two or three elements.

Reply:
Whoops 😅 thanks for the clarification